includes/classes/InternalConnections/NetworkSiteConnection.php

```diff
@@ -267,6 +267,12 @@ public function pull( $items ) {
 
 			restore_current_blog();
 
+			// Allow the sync'ed post to be updated via a REST request to
```

Comment:
@petenelson If you could change these code comments into an actual DocBlock, similar to how most of the other actions are documented (and that should follow https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters), that would be great